apps/web/components/shared/qa-client.tsx

```
@@ -0,0 +1,1200 @@
+          .from("comments")
+          .select("*")
+          .eq("project_id", projectId)
+          .not("parent_comment_id", "is", null) // Only get comments with a parent
+          .order("created_at", { ascending: true });
+
+        if (error) throw error;
+        
+        // If we have comments, fetch author data separately
+        if (data && data.length > 0) {
+          // Get unique author IDs
+          const authorIds = [...new Set(data.map((item: any) => item.author_id))];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)